operator/v1/types.go

```diff
@@ -70,11 +70,11 @@ type OperatorSpec struct {
 	// +kubebuilder:default=Normal
 	OperatorLogLevel LogLevel `json:"operatorLogLevel,omitempty"`
 
-	// unsupportedConfigOverrides holds a sparse config that will override any previously set options.  It only needs to be the fields to override
-	// it will end up overlaying in the following order:
-	// 1. hardcoded defaults
-	// 2. observedConfig
-	// 3. unsupportedConfigOverrides
```

Comment:
So I think the new content is good, useful context, but I feel like we are also losing some context by dropping this content, any way they can be combined?

Reply:
Thanks for the quick review and reply. I'll need to ask a few people.

Reply:
For context, here is the original comment for removing the original context: https://github.com/openshift/openshift-docs/pull/55319#discussion_r1114817459

I can see the reasoning to remove the original text, because of the unsupported nature of the resource.

Reply:
I think it can possibly be reworded, but pointing out that this overrides the `observedConfig` is worthwhile context IMO

Reply:
Thanks, @JoelSpeed . The text is reworded now.